unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneBoundariesController/Tests/SBC_Asserts.cs

```diff
@@ -70,7 +70,7 @@ public static IEnumerator NFTShapeIsInvalidatedWhenStartingOutOfBounds(ParcelSce
 
             var componentModel = new NFTShape.Model()
             {
-                src = "ethereum://0x06012c8cf97BEaD5deAe237070F9587f8E7A266d/558536"
+                src = "local://0x06012c8cf97BEaD5deAe237070F9587f8E7A266d/" + TestAssetsUtils.GetAbsolutePath() + "/NFT/nftInfo.json"
```

Comment:
I noticed you added a new piece of logic in `NFTShapeLoaderController` -> `FetchNFTImage()` only to make tests using local NFTs files and here you are changing the current tests from real NFTs urls to local ones, right?. But then, won't we have tests that contemplate real cases of valid/invalids urls? I mean for not "local" nfts?

Reply:
We should never test against external resources but I agree with you. To best way to solve this is to use mocking in the `NFTShapeController`.

I would go for something like a `INFTHelper` interface with a `FetchInfo` method. Since `NFTShapeController` is a `Monobehaviour` some trickery would be needed to inject the `INFTHelper`, the easiest way would be to just have it public and make the test override it.

Once we are able to mock the fetching of the NFT, we can just do whatever we want in the test: Force a failure, retrieve local asset as textures, etc..

We would also avoid having test code in production (the "local" protocol).

Reply:
Okey, I will change it.

I just copied the way we test the GLTF, since we have a code in the GLTF loader that test against local assets

Reply:
I agree with the general request. On addition, I'd like to add that `FileLoader` us an utility class that should be used exclusively by the GLTF importer, if we need something similar we should add that functionality to our web request controller. This way we keep the GLTF code decoupled from the rest of our repo as best as possible.

Reply:
I will rename the `FileLoader ` to `GLTFileLoader `so in the future we don't use this utility class for other use then